src/main/java/at/hannibal2/skyhanni/config/Storage.java

```diff
@@ -171,6 +171,12 @@ public static class PlotIcon {
 
             public static class Fortune {
 
+                @Expose
+                public Map<FarmingItems, Boolean> outdatedItems = new HashMap<>();
+
+                @Expose
+                public int uniqueVisitors = 0;
```

Comment:
Maybe store this outside of Fortune, directly in Garden. Will be useful for standalone gui element maybe?